modules/power/default.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.power;
+
+  onOff = cond: if cond then "On" else "Off";
```

Comment:
This produces `On` or `Off`, while the other two `onOff` implementations produce `on` or `off`. If this is deliberately different, please add a comment detailing why. Otherwise, let's keep them all the same

Reply:
Difference was not intentional. Fixed.